flasc/model_fitting/floris_tuning.py

```diff
@@ -159,12 +170,33 @@ def sweep_wd_std_for_er(
     df_freq=None,  # Not yet certain we will use this,
     remove_all_nulls=False,
 ):
-    """
-    Determine the best-fit wd_std for FLORIS by comparison with energy ratio plots
+    """Determine the best-fit wd_std for FLORIS by comparison with energy ratio plots.
 
     TODO: Reimplement that comparison only takes place when FLORIS value is below some threshold
-    """
 
+    Args:
+        value_candidates (list): The values to sweep
+        df_scada_in (DataFrame): The SCADA data
+        df_approx_ (DataFrame): The FLORIS approximation data
+        ref_turbines (list): The reference turbines
+        test_turbines (list): The test turbines
+        yaw_angles (np.ndarray): The yaw angles
+        wd_step (float): The wind direction step
+        wd_min (float): The minimum wind direction
+        wd_max (float): The maximum wind direction
+        ws_step (float): The wind speed step
+        ws_min (float): The minimum wind speed
+        ws_max (float): The maximum wind speed
+        bin_cols_in (list): The bin columns
+        weight_by (str): The weight method.  Can be 'min' or 'sum'.
+            Default is 'min'.
+        df_freq (DataFrame): The frequency data
+        remove_all_nulls (bool): Remove all nulls.  Default is False.
+
+    Returns:
+        (np.ndarray, list): The FLORIS energy ratio errors and the dataframes
```

Comment:
Again, maybe this should be a list of Returns. Maybe the google styling isn't specific about this though?

Reply:
I should've checked it earlier, it seems clear now to prefer to declare the output a tuple